example_configs/dolibarr.md

```diff
@@ -0,0 +1,88 @@
+# Configuration pour Dolibarr
+
+## This example will help you to create user in dolibarr from your users in your lldap server from a specific group and to login with the password from the lldap server.
```

Comment:
Let's not make this a title (remove the ##).

Reply:
I modified other lines to make titles to separate the three steps. Is that ok ?

Reply:
Almost :) You just need to make the first line a level 1 title (with only one #)